source/documentation/upgrade-guide/chap-Updates_between_Minor_Releases.html.md

```diff
@@ -58,6 +60,54 @@ Updates to the oVirt Engine are released via the oVirt Project.
 
 **Important:** The update process may take some time; allow time for the update process to complete and do not stop the process once initiated.
 
+## Updating the oVirt Self-Hosted Engine and Underlying Virtualization Host(s)
+
+The process for upgrading a [Self-Hosted Engine](/self-hosted/Self-Hosted_Engine_Guide/) is slightly different as the engine is running as a VM.
+
+The following assumes that you have already deployed the Hosted Engine on your hosts and the Hosted Engine VM is running the same oVirt version as the host(s).
+
+1.  Virtualization Host: Enable global maintenance mode from one of the hosts
+```
+host ~ # hosted-engine --set-maintenance --mode=global
+```
+
+2.  Virtualization Host: Update packages (you may need to changes repository to latest version or install the latest versions metapackage e.g. ovirt-release42-4.2.1)
```

Comment:
Aren't steps 2-4 already handled in another section here "Updating Virtualization Hosts", or if you prefer, "Manually Updating Hosts"? They suggest to reboot the host instead of restarting specific services, which I think is safer (and has better chances to be compatible with future versions, which might have different/new services).

If you agree, you can have a single procedure "Updating oVirt Engine", which has optional steps: "If you use hosted-engine, set global maintenance", ..., "If you use hosted-engine, exit global maintenance".

Reply:
And please note that in order to update packages on the host you have to explicitly put it into maintenance mode from the engine.
Updating virt related packages with VM in execution could kill them.